README.md

```diff
@@ -1,8 +1,21 @@
 # ember-simple-tree
 
-Simple tree component for Ember without any dependency.
+Lightweight, composable tree component for Ember without any dependency.
 
-## Installation
+This is based on ember-simple-tree with non-compatible changes to enable
+customizeability and the ability to compose and substitute your own
+components.
+
+
+Compatibility
+------------------------------------------------------------------------------
```

Comment:
Should use the pound sign instead to be consistent with the rest of the document

Reply:
done